src/ui/public/doc_table/doc_table.js

```diff
@@ -71,6 +71,12 @@ uiModules.get('kibana')
           if ($scope.columns.length === 0) $scope.columns.push('_source');
         });
 
+        const dispatchCustomEvent = (name) => {
```

Comment:
this gets repeated quite a few times ... extract somewhere everyone can access it ? maybe `vis.API.renderComplete()` ?